CMakeLists.txt

```diff
@@ -728,9 +728,9 @@ install(DIRECTORY "${openstudio_ROOT_DIR}/Python" DESTINATION "." COMPONENT "Pyt
 # install(DIRECTORY "${openstudio_ROOT_DIR}/lib" DESTINATION "." COMPONENT "OpenStudioApp" USE_SOURCE_PERMISSIONS)
 
 install(IMPORTED_RUNTIME_ARTIFACTS openstudio::openstudio DESTINATION bin COMPONENT "CLI")
-install(IMPORTED_RUNTIME_ARTIFACTS openstudio::openstudiolib DESTINATION bin COMPONENT "CLI")
-install(IMPORTED_RUNTIME_ARTIFACTS openstudio::rubyengine DESTINATION bin COMPONENT "CLI")
-install(IMPORTED_RUNTIME_ARTIFACTS openstudio::pythonengine DESTINATION bin COMPONENT "CLI")
+install(IMPORTED_RUNTIME_ARTIFACTS openstudio::openstudiolib DESTINATION lib COMPONENT "CLI")
```

Comment:
I think we want these in bin on Windows but lib on Linux/Mac

Reply:
I downloaded the installer and confirmed this should be bin on Windows.  The CLI wasn't working directly from the install.  I moved these three libs from the lib dir to bin dir and it worked in both the OS App and SketchUp